csfieldguide/static/interactives/matrix-simplifier/js/matrix-simplifier.js

```diff
@@ -186,7 +188,9 @@ function appendInput(type, inputHtml) {
   var $closeButton = $('<button type="button" class="close dismiss-eqtn" aria-label="Close">');
   $closeButton.append($('<span aria-hidden="true">&times;</span>'));
   var $editButton = $('<button type="button" class="close edit-eqtn" aria-label="Edit">');
+  var $moveLabel = $('<span class="close move-eqtn handle" aria-label="Move">=</span>');
```

Comment:
I think the equals sign looks too similar to a hamburger menu. Perhaps pick an icon from here? https://icons8.com/icons/set/drag